includes/sdk-ts-browser/shared-plugin-2-properties.md

```diff
@@ -0,0 +1,17 @@
+##### plugin.name [optional]
```

Comment:
I think it makes sense to callout at [here](https://pr-786.d19s7xzcva2mw3.amplifyapp.com/data/sdk-plugins/#enrichment-type-plugin_1) that we have some improvements for Browser 2.0.

Reply:
how do you recommend to callout the improvements? the improvements im seeing here mostly benefits the ones writing new plugins but don't see how it affects the current examples